sensor/common/networkflow/manager/purger.go

```diff
@@ -0,0 +1,167 @@
+package manager
+
+import (
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/stackrox/rox/pkg/concurrency"
+	"github.com/stackrox/rox/pkg/sync"
+	"github.com/stackrox/rox/pkg/timestamp"
+	flowMetrics "github.com/stackrox/rox/sensor/common/networkflow/metrics"
+)
+
+func (m *networkFlowManager) runPurger(tickerC <-chan time.Time, maxAge time.Duration) {
+	for {
+		select {
+		case <-m.stopper.Flow().StopRequested():
+			return
+		case <-tickerC:
+			npae := purgeActiveEndpoints(&m.activeEndpointsMutex, maxAge, m.activeEndpoints, m.clusterEntities)
+			npac := purgeActiveConnections(&m.activeConnectionsMutex, maxAge, m.activeConnections, m.clusterEntities)
+			nphe, nphc := purgeHostConns(&m.connectionsByHostMutex, maxAge, m.connectionsByHost, m.clusterEntities)
```

Comment:
nit: I'd rename these variables to something like `numActiveEndpointPurged`, etc.

Reply:
The smaller the scope of the variables in go, the more C-style variable names are okay - https://google.github.io/styleguide/go/decisions.html#variable-names. Here the scope is max 4 lines, so I thought it would be okay. 
But I can make it more explanatory, this is not an issue here.